dpgen/data/arginfo.py

```diff
@@ -35,6 +35,119 @@ def init_reaction_mdata_arginfo() -> Argument:
+    doc_from_poscar_path = "Path of POSCAR for VASP or STRU for ABACUS. **Necessary** if `from_poscar` is true."
+    doc_relax_incar = "Path of INCAR for VASP or INPUT for ABACUS for relaxation in VASP. **Necessary** if `stages` include 1."
+    doc_md_incar = "Path of INCAR for VASP or INPUT for ABACUS for MD in VASP. **Necessary** if `stages` include 3."
+    doc_scale = "Scales for transforming cells."
+    doc_skip_relax = "If it's true, you may directly run stage 2 (perturb and scale) using an unrelaxed POSCAR."
+    doc_pert_numb = "Number of perturbations for each POSCAR."
+    doc_pert_box = "Percentage of Perturbation for cells."
+    doc_pert_atom = "Perturbation of each atoms (Angstrom)."
+    doc_md_nstep = "Steps of AIMD in stage 3. If it's not equal to settings via `NSW` in `md_incar`, DP-GEN will follow `NSW`."
+    doc_coll_ndata = "Maximal number of collected data."
+    doc_type_map = "The indices of elements in deepmd formats will be set in this order."
+    doc_init_fp_style = "First-principle software. If this key is absent."
+    doc_relax_kpt = 'Path of `KPT` file for relaxation in stage 1. Only useful if `init_fp_style` is "ABACUS".'
+    doc_md_kpt = 'Path of `KPT` file for MD simulations in stage 3. Only useful if `init_fp_style` is "ABACUS".'
+    doc_atom_masses = 'List of atomic masses of elements. The order should be the same as `Elements`. Only useful if `init_fp_style` is "ABACUS".'
+    return Argument("init_bulk_jdata", dict, [
+        Argument("stages", list, optional=False, doc=doc_stages),
+        Argument("elements", list, optional=False, doc=doc_elements),
+        Argument("potcars", list, optional=True, doc=doc_potcars),
+        Argument("cell_type", str, optional=True, doc=doc_cell_type),
+        Argument("super_cell", list, optional=False, doc=doc_super_cell),
+        Argument("from_poscar", bool, optional=True, default=False, doc=doc_from_poscar),
+        Argument("from_poscar_path", str, optional=True, doc=doc_from_poscar_path),
+        Argument("relax_incar", str, optional=True, doc=doc_relax_incar),
+        Argument("md_incar", str, optional=True, doc=doc_md_incar),
+        Argument("scale", list, optional=False, doc=doc_scale),
+        Argument("skip_relax", bool, optional=False, doc=doc_skip_relax),
+        Argument("pert_numb", int, optional=False, doc=doc_pert_numb),
+        Argument("pert_box", float, optional=False, doc=doc_pert_box),
+        Argument("pert_atom", float, optional=False, doc=doc_pert_atom),
+        Argument("md_nstep", int, optional=False, doc=doc_md_nstep),
+        Argument("coll_ndata", int, optional=False, doc=doc_coll_ndata),
+        Argument("type_map", list, optional=True, doc=doc_type_map),
+        Argument("init_fp_style", str, optional=True, default="VASP", doc=doc_init_fp_style),
```

Comment:
Use [`Variant`](https://docs.deepmodeling.com/projects/dargs/en/stable/api/dargs.html#dargs.Variant) instead, and put `relax_kpt`, `md_kpt`, and `atom_masses` into the `Argument` of ABACUS style.